### DISCUSSION TITLE
Insert link to a file

### DISCUSSION THREAD

**Discussion by andreacap14** [PHPOffice/PhpSpreadsheet#2845] [Q&A]
I'm generating a script that generate and download a zip file containing an excel file and some pdf files.

I'm trying to do that in the excel in some cells the user can click on them and it should opens the linked pdf file.
Everything is working great except the link to the file (it simply writes the text withouth link)
Links to website is instead working great.
I'm using this code:

$spreadsheet->getActiveSheet()->getCellByColumnAndRow($actual_cloumn, $actual_row)->getHyperlink()->setUrl($pdf_filename);

How can I do that? Am I doing something wrong?

**Comment by MarkBaker** [comment]
What url are you using for the PDF if it's part of the downloaded zip?

What url would you use for the link if you had created the Excel file using MS Excel, and then put it in a zip file together with the pdf?
